audinterface/core/process.py

```diff
@@ -698,16 +772,33 @@ def __call__(
             self.channels,
             self.mixdown,
         )
-        if self.process_func_is_mono:
-            return [
-                self.process_func(
-                    np.atleast_2d(channel),
+
+        def _helper(x):
```

Comment:
Maybe call it `_process` instead of `_helper`, but on the other hand we have already a lot of functions/methods called process ;)

Reply:
OK, then let's stick with your current solution.